master/buildbot/test/integration/test_commandmixin.py

```diff
@@ -0,0 +1,96 @@
+# This file is part of Buildbot.  Buildbot is free software: you can
+# redistribute it and/or modify it under the terms of the GNU General Public
+# License as published by the Free Software Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright Buildbot Team Members
+from buildbot.process import results
+from buildbot.process.buildstep import BuildStep
+from buildbot.process.buildstep import CommandMixin
+from buildbot.test.util.integration import RunMasterBase
+
+from twisted.internet import defer
```

Comment:
Are you sure about this? All the integration tests have twisted import lines after buildbot, and when I swap them, I get tracebacks and test failures related to 'no such file or directory'.